en/docs/getting-started/basic-architecture.md

```diff
@@ -1,73 +1,72 @@
+[![Basic Architecture]({{base_path}}/assets/img/get_started/basic-architecture.png)]({{base_path}}/assets/img/get_started/basic-architecture.png)
 
 ## Management Plane
 
-The management plane is where API creation and API management takes place.  It consists of portals (publisher, developer portal, admin portal) for users to create and manage APIs, implement rate-limiting policies, monitor and monetization, etc. It also provides a set of APIs to interact with external tools like API Controller. The management plane includes API Analytics dashboards, displaying various business insights. . 
+The Management Plane is where API creation and API management takes place.  It consists of portals (Publisher, Developer Portal, Admin Portal) for users to create and manage APIs, implement rate limiting policies, monitor, and monetization, etc. It also provides a set of APIs to interact with external tools like API Controller. The Management Plane includes API Analytics dashboards, displaying various business insights.
 
 ### Components
 
 #### API Publisher
 
-![API Publisher]({{base_path}}/assets/img/learn/overviewpage-rest-api.jpg)
+[![API Publisher]({{base_path}}/assets/img/learn/overviewpage-rest-api.jpg)]({{base_path}}/assets/img/learn/overviewpage-rest-api.jpg)
 
-WSO2 API Manager’s API Publisher is a state-of-the-art GUI based tool for API development and management. The GUI is designed for API creators to develop, document, secure, test, and version APIs with ease. It’s also able to cater to more API management-related tasks such as publishing APIs, monetizing APIs, and applying rate-limiting policies.
+WSO2 API Manager’s API Publisher is a state-of-the-art GUI based tool for API development and management. The GUI is designed for API creators to develop, document, secure, test, and version APIs with ease. It’s also able to cater to more API management-related tasks such as publishing APIs, monetizing APIs, and applying rate limiting policies.
 
 #### API Developer Portal
 
-![Developer Portal]({{base_path}}/assets/img/get_started/developer-portal-overview.png)
-
-The Developer Portal is a state-of-the-art web interface that allows API publishers to host and advertise their APIs while allowing API consumers to self-register, discover, evaluate, subscribe to and consume APIs securely and easily.
+[![Developer Portal]({{base_path}}/assets/img/get_started/developer-portal-overview.png)]({{base_path}}/assets/img/get_started/developer-portal-overview.png)
 
+The Developer Portal is a state-of-the-art web interface that allows API publishers to host and advertise their APIs while allowing API consumers to self-register, discover, evaluate, subscribe to, and consume APIs securely and easily.
 
 #### API Analytics
 
-![API Analytics]({{base_path}}/assets/img/get_started/analytics.png)
+[![API Analytics]({{base_path}}/assets/img/get_started/analytics.png)]({{base_path}}/assets/img/get_started/analytics.png)
```

Comment:
<img width="733" alt="Screenshot 2020-12-09 at 17 35 11" src="https://user-images.githubusercontent.com/5195851/101627883-05d60a80-3a45-11eb-9dac-f81d38b40ed4.png">
It will be great if we can get the following updated in this image
Realtime > Real-Time
tcp connection > TCP connection
HTTP connection > HTTP connection